app/models/comfy/cms/page.rb

```diff
@@ -1,6 +1,9 @@
 require ComfortableMexicanSofa::Engine.root.join('app', 'models', 'comfy', 'cms', 'page.rb')
```

Comment:
It's because we are loading the page com comfortable mexican sofa engine. and reopened the class here in the model to our own customizations. A little play on the console:

```
[1] pry(main)> file = ComfortableMexicanSofa::Engine.root.join('app', 'models', 'comfy', 'cms', 'page')

=> #<Pathname:/Users/tomas-stefano/.rvm/gems/ruby-2.1.2@cms/gems/comfortable_mexican_sofa-1.12.1.64/app/models/comfy/cms/page>

[2] pry(main)> File.exist? file
=> false

[3] pry(main)> file = ComfortableMexicanSofa::Engine.root.join('app', 'models', 'comfy', 'cms', 'page.rb')

=> #<Pathname:/Users/tomas-stefano/.rvm/gems/ruby-2.1.2@cms/gems/comfortable_mexican_sofa-1.12.1.64/app/models/comfy/cms/page.rb>
[4] pry(main)> File.exist? file
=> true
```